### PR TITLE
REST Resources API to handle conditions being case insensitive

### PR DESCRIPTION
**Pull request by Sharvaribarve**
Changed named queries to handle queries in case insensitive way

Before you contribute, please review these guidelines to help ensure a smooth process for everyone.

Thanks.

* Read [how to properly contribute to open source projects on Github][1].
* Fork the project.
* Use a feature branch.
* Write [good commit messages][2].
* Use the same coding conventions as the rest of the project.
* Commit locally and push to your fork until you are happy with your contribution.
* Make sure to add tests and verify all the tests are passing when merging upstream.
* Add an entry to the [Changelog][3] accordingly.
* Please add your name to the [CONTRIBUTORS.md][7] file. Adding your name to the [CONTRIBUTORS.md][7] file signifies agreement to all rights and reservations provided by the [License][4].
* [Squash related commits together][5].
* Open a [pull request][6].
* The pull request will be reviewed by the community and merged by the project committers.

[1]: http://gun.io/blog/how-to-github-fork-branch-and-pull-request
[2]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
[3]: ./CHANGELOG.md
[4]: ./LICENSE
[5]: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html
[6]: https://help.github.com/articles/using-pull-requests
[7]: ./CONTRIBUTORS.md

### CODE REVIEW

**Review by PeterTeunissen — APPROVED** [review]

File: jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/JpaApplicationConfigTemplate.java [hunk 1]
```diff
@@ -13,26 +13,38 @@
 @Table(name = "APP_CONFIG_TEMPLATE", uniqueConstraints = {@UniqueConstraint(columnNames = {"APP_ID", "TEMPLATE_NAME", "JVM_ID"})})
 @NamedQueries({
         @NamedQuery(name = JpaApplicationConfigTemplate.GET_APP_RESOURCE_TEMPLATE_NAMES,
-                query = "SELECT DISTINCT t.templateName FROM JpaApplicationConfigTemplate t WHERE t.app.name = :appName and t.jvm.name = :jvmName"),
+                query = "SELECT DISTINCT t.templateName FROM JpaApplicationConfigTemplate t WHERE LOWER(t.app.name) =" +
+                        " " +
```
Comment (jmahmoodcc) [inline jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/JpaApplicationConfigTemplate.java]: Clean up this empty line

File: jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/JpaApplicationConfigTemplate.java [hunk 2]
```diff
@@ -13,26 +13,38 @@
...
+                query = "SELECT t.templateContent FROM JpaApplicationConfigTemplate t where LOWER(t.app.name) = " +
+                        "LOWER(:appName) and t.templateName = :templateName and t.jvm = :templateJvm"),
         @NamedQuery(name = JpaApplicationConfigTemplate.GET_APP_TEMPLATE_META_DATA,
-                query = "SELECT t.metaData FROM JpaApplicationConfigTemplate t where t.app.name = :appName and t.templateName = :templateName and t.jvm = :templateJvm"),
+                query = "SELECT t.metaData FROM JpaApplicationConfigTemplate t where LOWER(t.app.name) = " +
+                        "LOWER(:appName) and t.templateName = :templateName and t.jvm = :templateJvm"),
         @NamedQuery(name = JpaApplicationConfigTemplate.UPDATE_APP_TEMPLATE_CONTENT,
-                query = "UPDATE JpaApplicationConfigTemplate t SET t.templateContent = :templateContent WHERE t.app.name = :appName AND t.templateName = :templateName and t.jvm = :templateJvm"),
+                query = "UPDATE JpaApplicationConfigTemplate t SET t.templateContent = :templateContent WHERE LOWER(t" +
```
Comment (jmahmoodcc) [inline jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/JpaApplicationConfigTemplate.java]: Don't break up this variable across lines

File: jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/JpaApplicationConfigTemplate.java [hunk 3]
```diff
@@ -13,26 +13,38 @@
...
+                        ".app.name) = LOWER(:appName) AND t.templateName = :templateName and t.jvm = " +
+                        ":templateJvm"),
         @NamedQuery(name = JpaApplicationConfigTemplate.UPDATE_APP_TEMPLATE_META_DATA,
-                query = "UPDATE JpaApplicationConfigTemplate t SET t.metaData = :metaData WHERE t.app.name = :appName AND t.templateName = :templateName and t.jvm = :templateJvm"),
+                query = "UPDATE JpaApplicationConfigTemplate t SET t.metaData = :metaData WHERE LOWER(t.app.name) = " +
+                        "LOWER(:appName) AND t.templateName = :templateName and t.jvm = :templateJvm"),
         @NamedQuery(name = JpaApplicationConfigTemplate.GET_APP_TEMPLATE,
-                query = "SELECT t FROM JpaApplicationConfigTemplate t where t.templateName = :tempName and t.app.name = :appName and t.jvm.name = :jvmName"),
+                query = "SELECT t FROM JpaApplicationConfigTemplate t where t.templateName = :tempName and LOWER(t" +
```
Comment (jmahmoodcc) [inline jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/JpaApplicationConfigTemplate.java]: Don't break up this variable across lines

File: jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/JpaApplicationConfigTemplate.java [hunk 4]
```diff
@@ -13,26 +13,38 @@
...
+                        ".app.name) = LOWER(:appName) and LOWER(t.jvm.name) = LOWER(:jvmName)"),
         @NamedQuery(name = JpaApplicationConfigTemplate.QUERY_DELETE_APP_TEMPLATE, query = "DELETE FROM JpaApplicationConfigTemplate t WHERE t.templateName = :templateName"),
         @NamedQuery(name = JpaApplicationConfigTemplate.GET_APP_TEMPLATE_RESOURCE_NAME,
-                query = "SELECT t.templateName FROM JpaApplicationConfigTemplate t WHERE t.app.name = :appName AND t.templateName = :templateName"),
+                query = "SELECT t.templateName FROM JpaApplicationConfigTemplate t WHERE LOWER(t.app.name) = " +
+                        "LOWER(:appName) AND t.templateName = :templateName"),
         @NamedQuery(name = JpaApplicationConfigTemplate.QUERY_DELETE_APP_RESOURCE_BY_TEMPLATE_APP_JVM_NAME,
-                query = "DELETE FROM JpaApplicationConfigTemplate t WHERE t.templateName = :templateName AND t.app.name = :appName AND t.jvm.name = :jvmName"),
+                query = "DELETE FROM JpaApplicationConfigTemplate t WHERE t.templateName = :templateName AND LOWER(t" +
```
Comment (jmahmoodcc) [inline jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/JpaApplicationConfigTemplate.java]: Don't break up this variable across lines

File: jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/JpaApplicationConfigTemplate.java [hunk 5]
```diff
@@ -13,26 +13,38 @@
...
+                        ".app.name) = LOWER(:appName) AND LOWER(t.jvm.name) = LOWER(:jvmName)"),
         @NamedQuery(name = JpaApplicationConfigTemplate.QUERY_DELETE_APP_RESOURCES_BY_TEMPLATE_NAME_LIST_APP_JVM_NAME,
-                query = "DELETE FROM JpaApplicationConfigTemplate t WHERE t.templateName IN :templateNameList AND t.app.name = :appName AND t.jvm.name = :jvmName"),
+                query = "DELETE FROM JpaApplicationConfigTemplate t WHERE t.templateName IN :templateNameList AND " +
+                        "LOWER(t.app.name) = LOWER(:appName) AND LOWER(t.jvm.name) = LOWER(:jvmName)"),
         @NamedQuery(name = JpaApplicationConfigTemplate.QUERY_GET_APP_RESOURCE,
-                query = "SELECT t FROM JpaApplicationConfigTemplate t where t.app.name = :appName and t.templateName = :templateName and t.jvm.name = :jvmName"),
+                query = "SELECT t FROM JpaApplicationConfigTemplate t where LOWER(t.app.name) = LOWER(:appName) and t" +
```
Comment (jmahmoodcc) [inline jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/JpaApplicationConfigTemplate.java]: Don't break up this variable across lines

File: jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaGroupAppConfigTemplate.java [hunk 6]
```diff
@@ -12,24 +12,36 @@
...
+                query = "SELECT t.templateContent FROM JpaGroupAppConfigTemplate t where LOWER(t.grp.name) = " +
+                        "LOWER(:grpName) and LOWER(t.app.name) = LOWER(:appName) and t.templateName = :templateName"),
         @NamedQuery(name = JpaGroupAppConfigTemplate.GET_GROUP_APP_TEMPLATE_META_DATA,
-                query = "SELECT t.metaData FROM JpaGroupAppConfigTemplate t where t.grp.name = :grpName and t.templateName = :templateName"),
+                query = "SELECT t.metaData FROM JpaGroupAppConfigTemplate t where LOWER(t.grp.name) = LOWER(:grpName)" +
+                        " and t.templateName = :templateName"),
         @NamedQuery(name = JpaGroupAppConfigTemplate.UPDATE_GROUP_APP_TEMPLATE_CONTENT,
-                query = "UPDATE JpaGroupAppConfigTemplate t SET t.templateContent = :templateContent WHERE t.grp.name = :grpName AND t.app.name = :appName AND t.templateName = :templateName"),
+                query = "UPDATE JpaGroupAppConfigTemplate t SET t.templateContent = :templateContent WHERE LOWER(t" +
```
Comment (jmahmoodcc) [inline jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaGroupAppConfigTemplate.java]: Don't break up this variable across lines

File: jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaGroupJvmConfigTemplate.java [hunk 7]
```diff
@@ -11,23 +11,33 @@
...
-                query = "SELECT t.templateContent FROM JpaGroupJvmConfigTemplate t where t.grp.name = :grpName and t.templateName = :templateName"),
+                query = "SELECT t.templateContent FROM JpaGroupJvmConfigTemplate t where LOWER(t.grp.name) = " +
+                        "LOWER(:grpName) and t.templateName = :templateName"),
         @NamedQuery(name = JpaGroupJvmConfigTemplate.GET_GROUP_JVM_TEMPLATE_META_DATA,
-                query = "SELECT t.metaData FROM JpaGroupJvmConfigTemplate t where t.grp.name = :grpName and t.templateName = :templateName"),
+                query = "SELECT t.metaData FROM JpaGroupJvmConfigTemplate t where LOWER(t.grp.name) = LOWER(:grpName)" +
+                        " and t.templateName = :templateName"),
         @NamedQuery(name = JpaGroupJvmConfigTemplate.UPDATE_GROUP_JVM_TEMPLATE_CONTENT,
-                query = "UPDATE JpaGroupJvmConfigTemplate t SET t.templateContent = :templateContent WHERE t.grp.name = :grpName AND t.templateName = :templateName"),
+                query = "UPDATE JpaGroupJvmConfigTemplate t SET t.templateContent = :templateContent WHERE LOWER(t" +
```
Comment (jmahmoodcc) [inline jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaGroupJvmConfigTemplate.java]: Don't break up this variable across lines

File: jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaGroupJvmConfigTemplate.java [hunk 8]
```diff
@@ -11,23 +11,33 @@
...
+                        "LOWER(:grpName) AND t.templateName = :templateName"),
         @NamedQuery(name = JpaGroupJvmConfigTemplate.QUERY_DELETE_GRP_JVM_TEMPLATE, query = "DELETE FROM JpaGroupJvmConfigTemplate t WHERE t.templateName = :templateName"),
         @NamedQuery(name = JpaGroupJvmConfigTemplate.GET_GROUP_JVM_TEMPLATE_RESOURCE_NAME,
-                query = "SELECT t.templateName FROM JpaGroupJvmConfigTemplate t WHERE t.grp.name = :grpName AND t.templateName = :templateName"),
-        @NamedQuery(name = JpaGroupJvmConfigTemplate.QUERY_DELETE_GROUP_LEVEL_JVM_RESOURCE_BY_TEMPLATE_GROUP_NAME, query = "DELETE FROM JpaGroupJvmConfigTemplate t WHERE t.templateName = :templateName AND t.grp.name = :grpName"),
+                query = "SELECT t.templateName FROM JpaGroupJvmConfigTemplate t WHERE LOWER(t.grp.name) = " +
+                        "LOWER(:grpName) AND t.templateName = :templateName"),
+        @NamedQuery(name = JpaGroupJvmConfigTemplate.QUERY_DELETE_GROUP_LEVEL_JVM_RESOURCE_BY_TEMPLATE_GROUP_NAME,
+                query = "DELETE FROM JpaGroupJvmConfigTemplate t WHERE t.templateName = :templateName AND LOWER(t.grp" +
```
Comment (jmahmoodcc) [inline jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaGroupJvmConfigTemplate.java]: Don't break up this variable across lines

File: jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaGroupJvmConfigTemplate.java [hunk 9]
```diff
@@ -11,23 +11,33 @@
...
+                        ".name) = LOWER(:grpName)"),
         @NamedQuery(name = JpaGroupJvmConfigTemplate.QUERY_DELETE_GROUP_LEVEL_JVM_RESOURCES_BY_TEMPLATE_NAME_LIST_GROUP_NAME,
-                    query = "DELETE FROM JpaGroupJvmConfigTemplate t WHERE t.templateName IN :templateNameList AND t.grp.name = :grpName"),
+                    query = "DELETE FROM JpaGroupJvmConfigTemplate t WHERE t.templateName IN :templateNameList AND " +
+                            "LOWER(t.grp.name) = LOWER(:grpName)"),
         @NamedQuery(name = JpaGroupJvmConfigTemplate.QUERY_GET_GROUP_LEVEL_JVM_RESOURCE,
-                query = "SELECT t FROM JpaGroupJvmConfigTemplate t where t.grp.name = :grpName and t.templateName = :templateName")
+                query = "SELECT t FROM JpaGroupJvmConfigTemplate t where LOWER(t.grp.name) = LOWER(:grpName) and t" +
```
Comment (jmahmoodcc) [inline jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaGroupJvmConfigTemplate.java]: Don't break up this variable across lines

File: jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaGroupWebServerConfigTemplate.java [hunk 10]
```diff
@@ -11,23 +11,34 @@
...
         @NamedQuery(name = JpaGroupWebServerConfigTemplate.UPDATE_GROUP_WEBSERVER_TEMPLATE_META_DATA,
-                query = "UPDATE JpaGroupWebServerConfigTemplate t SET t.metaData = :metaData WHERE t.grp.name = :grpName AND t.templateName = :templateName"),
+                query = "UPDATE JpaGroupWebServerConfigTemplate t SET t.metaData = :metaData WHERE LOWER(t.grp.name) " +
+                        "= LOWER(:grpName) AND t.templateName = :templateName"),
         @NamedQuery(name = JpaGroupWebServerConfigTemplate.QUERY_DELETE_GRP_WEBSERVER_TEMPLATE, query = "DELETE FROM JpaGroupWebServerConfigTemplate t WHERE t.templateName = :templateName"),
-        @NamedQuery(name = JpaGroupWebServerConfigTemplate.GET_GROUP_WEBSERVER_TEMPLATE_RESOURCE_NAME, query = "SELECT t.templateName FROM JpaGroupWebServerConfigTemplate t WHERE t.grp.name = :grpName and t.templateName = :templateName"),
+        @NamedQuery(name = JpaGroupWebServerConfigTemplate.GET_GROUP_WEBSERVER_TEMPLATE_RESOURCE_NAME, query =
+                "SELECT t.templateName FROM JpaGroupWebServerConfigTemplate t WHERE LOWER(t.grp.name) = LOWER" +
```
Comment (jmahmoodcc) [inline jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaGroupWebServerConfigTemplate.java]: Don't break up this variable across lines

File: jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaGroupWebServerConfigTemplate.java [hunk 11]
```diff
@@ -11,23 +11,34 @@
...
+                query = "DELETE FROM JpaGroupWebServerConfigTemplate t WHERE t.templateName = :templateName AND " +
+                        "LOWER(t.grp.name) = LOWER(:grpName)"),
         @NamedQuery(name = JpaGroupWebServerConfigTemplate.QUERY_DELETE_GROUP_LEVEL_WEBSERVER_RESOURCES_BY_TEMPLATE_NAME_LIST_GROUP_NAME,
-                query = "DELETE FROM JpaGroupWebServerConfigTemplate t WHERE t.templateName IN :templateNameList AND t.grp.name = :grpName"),
+                query = "DELETE FROM JpaGroupWebServerConfigTemplate t WHERE t.templateName IN :templateNameList AND " +
+                        "LOWER(t.grp.name) = LOWER(:grpName)"),
         @NamedQuery(name = JpaGroupWebServerConfigTemplate.QUERY_GET_GROUP_LEVEL_WEBSERVER_RESOURCE,
-                query = "SELECT t FROM JpaGroupWebServerConfigTemplate t where t.grp.name = :grpName and t.templateName = :templateName"),
+                query = "SELECT t FROM JpaGroupWebServerConfigTemplate t where LOWER(t.grp.name) = LOWER(:grpName) " +
+                        "and t" +
```
Comment (jmahmoodcc) [inline jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaGroupWebServerConfigTemplate.java]: Don't break up this variable across lines

File: jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaJvmConfigTemplate.java [hunk 12]
```diff
@@ -13,24 +13,35 @@
 @Table(name = "JVM_CONFIG_TEMPLATE", uniqueConstraints = {@UniqueConstraint(columnNames = {"JVM_ID", "TEMPLATE_NAME"})})
 @NamedQueries({
         @NamedQuery(name = JpaJvmConfigTemplate.GET_JVM_RESOURCE_TEMPLATE_NAMES,
-                query = "SELECT t.templateName FROM JpaJvmConfigTemplate t WHERE t.jvm.name = :jvmName"),
+                query = "SELECT t.templateName FROM JpaJvmConfigTemplate t WHERE LOWER(t.jvm.name) = LOWER(:jvmName)"),
         @NamedQuery(name = JpaJvmConfigTemplate.GET_JVM_TEMPLATE_CONTENT,
-                query = "SELECT t.templateContent FROM JpaJvmConfigTemplate t where t.jvm.name = :jvmName and t.templateName = :templateName"),
+                query = "SELECT t.templateContent FROM JpaJvmConfigTemplate t where LOWER(t.jvm.name) = LOWER" +
```
Comment (jmahmoodcc) [inline jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaJvmConfigTemplate.java]: Don't break up this variable across lines

File: jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaJvmConfigTemplate.java [hunk 13]
```diff
@@ -13,24 +13,35 @@
...
+                query = "SELECT t.metaData FROM JpaJvmConfigTemplate t where LOWER(t.jvm.name) = LOWER(:jvmName) and " +
+                        "t.templateName = :templateName"),
         @NamedQuery(name = JpaJvmConfigTemplate.UPDATE_JVM_TEMPLATE_META_DATA,
-                query = "UPDATE JpaJvmConfigTemplate t SET t.metaData= :metaData WHERE t.jvm.name = :jvmName AND t.templateName = :templateName"),
+                query = "UPDATE JpaJvmConfigTemplate t SET t.metaData= :metaData WHERE LOWER(t.jvm.name) = " +
+                        "LOWER(:jvmName) AND " +
+                        "t.templateName = :templateName"),
         @NamedQuery(name = JpaJvmConfigTemplate.UPDATE_JVM_TEMPLATE_CONTENT,
-                query = "UPDATE JpaJvmConfigTemplate t SET t.templateContent = :templateContent WHERE t.jvm.name = :jvmName AND t.templateName = :templateName"),
+                query = "UPDATE JpaJvmConfigTemplate t SET t.templateContent = :templateContent WHERE LOWER(t.jvm" +
```
Comment (jmahmoodcc) [inline jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaJvmConfigTemplate.java]: Don't break up this variable across lines

File: jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaWebServerConfigTemplate.java [hunk 14]
```diff
@@ -13,28 +13,40 @@
...
+                query = "SELECT t.templateContent FROM JpaWebServerConfigTemplate t where LOWER(t.webServer.name) = " +
+                        "LOWER(:webServerName) and t.templateName = :templateName"),
         @NamedQuery(name = JpaWebServerConfigTemplate.GET_WEBSERVER_TEMPLATE_METADATA,
-                query = "SELECT t.metaData FROM JpaWebServerConfigTemplate t where t.webServer.name = :webServerName and t.templateName = :templateName"),
+                query = "SELECT t.metaData FROM JpaWebServerConfigTemplate t where LOWER(t.webServer.name) = " +
+                        "LOWER(:webServerName) and t.templateName = :templateName"),
         @NamedQuery(name = JpaWebServerConfigTemplate.UPDATE_WEBSERVER_TEMPLATE_CONTENT,
-                query = "UPDATE JpaWebServerConfigTemplate t SET t.templateContent = :templateContent WHERE t.webServer.name = :webServerName AND t.templateName = :templateName"),
+                query = "UPDATE JpaWebServerConfigTemplate t SET t.templateContent = :templateContent WHERE LOWER(t" +
```
Comment (jmahmoodcc) [inline jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaWebServerConfigTemplate.java]: Don't break up this variable across lines

File: jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaWebServerConfigTemplate.java [hunk 15]
```diff
@@ -13,28 +13,40 @@
...
+                query = "UPDATE JpaWebServerConfigTemplate t SET t.metaData= :metaData WHERE LOWER(t.webServer.name) " +
+                        "= LOWER(:webServerName) AND t.templateName = :templateName"),
         @NamedQuery(name = JpaWebServerConfigTemplate.GET_WEBSERVER_TEMPLATE,
-        query = "SELECT t FROM JpaWebServerConfigTemplate t where t.webServer.name = :webServerName and t.templateName = :templateName"),
+        query = "SELECT t FROM JpaWebServerConfigTemplate t where LOWER(t.webServer.name) = LOWER(:webServerName) and" +
+                " t.templateName = :templateName"),
         @NamedQuery(name = JpaWebServerConfigTemplate.QUERY_DELETE_WEB_SERVER_TEMPLATE, query="DELETE FROM JpaWebServerConfigTemplate t WHERE t.templateName = :templateName"),
         @NamedQuery(name = JpaWebServerConfigTemplate.QUERY_DELETE_WEBSERVER_RESOURCE_BY_TEMPLATE_WEBSERVER_NAME,
-                    query="DELETE FROM JpaWebServerConfigTemplate t WHERE t.templateName = :templateName AND t.webServer.name = :webServerName"),
+                    query="DELETE FROM JpaWebServerConfigTemplate t WHERE t.templateName = :templateName AND LOWER(t" +
```
Comment (jmahmoodcc) [inline jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaWebServerConfigTemplate.java]: Don't break up this variable across lines

File: jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaWebServerConfigTemplate.java [hunk 16]
```diff
@@ -13,28 +13,40 @@
...
-        @NamedQuery(name = JpaWebServerConfigTemplate.GET_WEBSERVER_TEMPLATE_RESOURCE_NAME, query = "SELECT t FROM JpaWebServerConfigTemplate t WHERE t.templateName = :templateName AND t.webServer.name = :webServerName"),
+                query = "SELECT t FROM JpaWebServerConfigTemplate t WHERE LOWER(t.webServer.name) = " +
+                        "LOWER(:webServerName)"),
+        @NamedQuery(name = JpaWebServerConfigTemplate.GET_WEBSERVER_TEMPLATE_RESOURCE_NAME, query = "SELECT t FROM " +
+                "JpaWebServerConfigTemplate t WHERE t.templateName = :templateName AND LOWER(t.webServer.name) = " +
+                "LOWER(:webServerName)"),
 
         @NamedQuery(name = JpaWebServerConfigTemplate.QUERY_DELETE_WEBSERVER_RESOURCES_BY_TEMPLATE_NAME_LIST_WEBSERVER_NAME,
-                query="DELETE FROM JpaWebServerConfigTemplate t WHERE t.templateName IN :templateNameList AND t.webServer.name = :webServerName"),
+                query="DELETE FROM JpaWebServerConfigTemplate t WHERE t.templateName IN :templateNameList AND LOWER(t" +
```
Comment (jmahmoodcc) [inline jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaWebServerConfigTemplate.java]: Don't break up this variable across lines

File: jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaWebServerConfigTemplate.java [hunk 17]
```diff
@@ -13,28 +13,40 @@
...
+                        ".webServer.name)= LOWER(:webServerName)"),
         @NamedQuery(name = JpaWebServerConfigTemplate.QUERY_GET_WEBSERVER_RESOURCE,
-                query = "SELECT t FROM JpaWebServerConfigTemplate t WHERE t.templateName = :templateName AND t.webServer.name = :webServerName")
+                query = "SELECT t FROM JpaWebServerConfigTemplate t WHERE t.templateName = :templateName AND LOWER(t" +
```
Comment (jmahmoodcc) [inline jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaWebServerConfigTemplate.java]: Don't break up this variable across lines

**Review by jmahmoodcc — CHANGES_REQUESTED** [review]
Looks good, just some minor formatting changes. I'm sure you used the auto-formatter but it looks like it broke up a bunch of the queries in strange places. Readability counts!

**Comment by jmahmoodcc** [comment]
Make sure to run the postman tests on your build just to make sure all the query changes were done correctly.

**Review by jmahmoodcc — APPROVED** [review]

**Pull request merged**